io_framework/csv_to_dataframe.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import os
+from resources.config import RESOURCES_DIR
+
+
+def csv_to_dataframe(filepath, row_start, row_end):
+    """
+    The parameters to convert a csv file into a dataframe.
+    :param filepath: Location of the file in a directory(dependent on linux and windows file systems)
+    :param row_start: A row to start reading data from
+    :param row_end: What row to end on
+    :return: dataframe
+
+    """
+    df = pd
+    count = 0
+    copyfilepath = RESOURCES_DIR + "/" + 'copytemp.csv'
+    with open(filepath, 'r') as f:
```

Comment:
Replace opening the file with **f = open('workfile', 'w')** syntax to make this more readable

Reply:
I don't think we need that because "with" keyword is also specifies that after finishing the block of code within the file will be closed. That's Python way of handling files :)

Reply:
Don't like the indentation. But sure, I guess haha